fed/mosaic-sns/src/main/java/org/eclipse/mosaic/fed/sns/ambassador/SnsAmbassador.java

```diff
@@ -127,30 +128,30 @@ private void process(RsuRegistration interaction) {
         final RsuMapping applicationRsu = interaction.getMapping();
         if (applicationRsu.hasApplication()) {
             SimulationEntities.INSTANCE.createOrUpdateOfflineNode(applicationRsu.getName(), applicationRsu.getPosition().toCartesian());
-            log.info("Added RSU id={} @time={}", applicationRsu.getName(), TIME.format(interaction.getTime()));
+            log.info("Added RSU id={} position={} @time={}", applicationRsu.getName(), applicationRsu.getPosition(), TIME.format(interaction.getTime()));
         }
     }
 
     private void process(TrafficLightRegistration interaction) {
         final TrafficLightMapping applicationTl = interaction.getMapping();
         if (applicationTl.hasApplication()) {
             SimulationEntities.INSTANCE.createOrUpdateOfflineNode(applicationTl.getName(), applicationTl.getPosition().toCartesian());
-            log.info("Added TrafficLight id={} @time={}", applicationTl.getName(), TIME.format(interaction.getTime()));
+            log.info("Added TrafficLight id={} position={} @time={}", applicationTl.getName(), applicationTl.getPosition(), TIME.format(interaction.getTime()));
         }
     }
 
     private void process(ChargingStationRegistration interaction) {
         final ChargingStationMapping applicationCs = interaction.getMapping();
         if (applicationCs.hasApplication()) {
             SimulationEntities.INSTANCE.createOrUpdateOfflineNode(applicationCs.getName(), applicationCs.getPosition().toCartesian());
-            log.info("Added ChargingStation id={} @time={}", applicationCs.getName(), TIME.format(interaction.getTime()));
+            log.info("Added ChargingStation id={} position={} @time={}", applicationCs.getName(), applicationCs.getPosition(), TIME.format(interaction.getTime()));
         }
     }
 
     private void process(VehicleUpdates interaction) {
         for (VehicleData added : interaction.getAdded()) {
             if (addOrUpdateVehicle(added)) {
-                log.info("Added Vehicle id={} @time={}", added.getName(), TIME.format(interaction.getTime()));
+                log.info("Added Vehicle id={} position={} @time={}", added.getName(), added.getPosition(), TIME.format(interaction.getTime()));
```

Comment:
You should move this line to `addOrUpdateVehicle` where the method `createOnlineNode` is called, I guess

Reply:
But that's exactly the case we're not interested in. We don't care for vehicles which are never considered by the communication simulation (e.g. background traffic).

Reply:
Oh, the case I mentioned was not applicable, here, sorry.

Reply:
@fhilg Have you worked on this? This line is currently missing in the log files.

The line should be move to `addOrUpdateVehicle` where the method `createOnlineNode` is called

Reply:
No, I haven't moved this line because
- it was a guess, and
- there was no decision made in our last meeting.

Reply:
> there was no decision made in our last meeting.

It actually was decided ;) maybe it was not too clear. Please just do what I've proposed and see if it works